src/test/java/org/isf/patient/rest/PatientControllerTest.java

```diff
@@ -142,16 +150,15 @@ public void when_post_patients_is_call_with_empty_body_then_BadRequest_HttpMessa
 		String empty_body = "";
 
 		MvcResult result = this.mockMvc
-				.perform(
-						post(request)
-								.content(empty_body.getBytes())
-								.contentType(MediaType.APPLICATION_JSON)
-				)
-				.andDo(log())
-				.andExpect(status().is4xxClientError())
-				.andExpect(status().isBadRequest())
-				.andExpect(content().string(anyOf(nullValue(), equalTo(""))))
-				.andReturn();
+						.perform(
```

Comment:
This indentation seems excessive....

Reply:
I used the indentation of OH, but I change it manualy

Reply:
There are too many occurencies, can we modify the OH indentation file settings?

Reply:
![Screenshot from 2023-04-21 21-58-34](https://user-images.githubusercontent.com/6343630/233724199-359f6cbf-b8f4-4eed-a316-d4ec30faa359.png)

Reply:
I tried to format too that part and I don't see changes, looks like GitHub spacing looks different but the code is correct

![image](https://user-images.githubusercontent.com/2938553/236054653-18e23bb7-83d4-4672-b57a-9e27a2de3fe7.png)